gen-web/src/index.js

```diff
@@ -371,14 +414,14 @@
   * Step Listener to initiate step specific actions
   */
   const constantCheck = () => {
-    if (stepTracker === 2) {
-      /* Add click listener to page container on Page 2 form intake */
-      inlineVariables.contentContainer.onclick = verifyInputData
```

Comment:
Not sure what the purpose of this code was. Removing it fixes an issue where while the next step is loading, `buttons.nextStep.disabled` is set to `true`, but it can get overwritten to `false` by this call to `verifyInputData`.